app/scripts/lib/resolver.js

```diff
@@ -61,7 +61,7 @@ function getRegistrar (type) {
 module.exports.resolve = function (name, provider) {
   const path = name.split('.')
   const tld = path[path.length - 1]
-  if (tld === 'eth') {
+  if (tld === 'eth' || tld === 'test') {
```

Comment:
what is tld?

Reply:
TLD === "top-level domain" ([reference](https://en.wikipedia.org/wiki/Top-level_domain)). I updated the variable name so it's not as cryptic.